rllib/examples/vizdoom_with_attention_net.py

```diff
@@ -73,14 +73,17 @@
         "episode_reward_mean": args.stop_reward,
     }
 
-    results = tune.run(
+    results = tune.Tuner(
         args.run,
-        config=config,
-        stop=stop,
-        verbose=2,
-        checkpoint_freq=5,
-        checkpoint_at_end=True,
-        restore=args.from_checkpoint,
```

Comment:
I guess we don't support the restore argument without the RLTrainer - should we remove `args.from_checkpoint` or maybe pass the tuner kwargs?

Reply:
Yes, that's correct.
Prefer to just remove it. (I think I don't have lingering `args.from_checkpoint`? Or do you see any?)

Reply:
Ah I think it's in line 15